frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -82,6 +83,14 @@ const EnterprisePlanBadge = () => (
     </Tooltip>
 );
 
+const listItemButtonStyle = (theme: Theme) => ({
```

Comment:
<img width="206" alt="Screenshot 2024-05-27 at 13 17 13" src="https://github.com/Unleash/unleash/assets/1394682/1c206c99-5bbb-4090-99eb-68072d5fb46f">
<img width="157" alt="Screenshot 2024-05-27 at 13 17 20" src="https://github.com/Unleash/unleash/assets/1394682/9b1445ab-aa2d-45a6-a8a5-5c0f12be36d5">

Border color and rounded corners. Using as mixing since TS doesn't allow to style `ListItemsButton component={Link}`